redisson/src/test/java/org/redisson/RedissonStreamTest.java

```diff
@@ -520,6 +520,31 @@ public void testReadGroup() {
         assertThat(s2).isEmpty();
     }
     
+    @Test
+    public void testAutogenerateStreamSequenceId() {
+        RStream<String, String> stream = redisson.getStream("test");
+        assertThat(stream.size()).isEqualTo(0);
+        
+        StreamMessageId id = new StreamMessageId(1).autogenerateSequenceId();
+        
+        Map<String, String> entry1 = new HashMap<>();
+        entry1.put("test", "value1");
+        Map<String, String> entry2 = new HashMap<>();
+        entry2.put("test", "value2");
+
+        stream.add(id,StreamAddArgs.entries(entry1));
+        stream.add(id,StreamAddArgs.entries(entry2));
+        
+        Map<StreamMessageId, Map<String, String>> r = stream.range(10, StreamMessageId.MIN, StreamMessageId.MAX);
+        System.out.println("r:" + r);
```

Comment:
system.out can't be used

Reply:
There some more like this in this test:
https://github.com/redisson/redisson/blob/018fbdfc9bebc29d77e3d8e006ff21000b637f7b/redisson/src/test/java/org/redisson/RedissonStreamTest.java#L162

I'll remove mine right away!